README.rst

```diff
@@ -66,10 +66,30 @@ pkgdb2::
 
     (my-pkgdb2-env)$ python createdb.py
 
+Setting up PostgreSQL
+=====================
+You can set up PostgreSQL instead of sqlite
```

Comment:
So we should give user an option kind of thing ?  I think with Postgres working with databump is a lot easire . :) So what should we put here?